Zend/zend_compile.c

```diff
@@ -2305,10 +2310,20 @@ static void zend_short_circuiting_commit(uint32_t checkpoint, znode *result, zen
 	}
 }
 
-static void zend_emit_jmp_null(znode *obj_node)
+static void zend_emit_jmp_null(znode *obj_node, zend_bool delay)
```

Comment:
You could just use ``bool`` as the Zend Engine 4 typedefs ``zend_bool`` to ``bool``

Reply:
Ok, I wasn't aware of that change 👍